pb/p2pd.proto

```diff
@@ -125,4 +128,42 @@ message ConnManagerRequest {
 
 message DisconnectRequest {
   required bytes peer = 1;
+}
+
+message PSRequest {
+  enum Type {
+    GET_TOPICS = 0;
+    LIST_PEERS = 1;
+    PUBLISH    = 2;
+    SUBSCRIBE  = 3;
+    CANCEL     = 4;
+    NEXT       = 5;
+  }
+
+  required Type type = 1;
+  optional string topic = 2;
+  optional bytes data = 3;
+  optional int64 timeout = 4;
```

Comment:
this is unnecessary -- there is no operation in pubsub that takes a timeout.

Reply:
My mistake, I thought I saw one. Removed